src/components/Table/index.js

```diff
@@ -1,10 +1,31 @@
 import React from "react";
 
 import { StyleTitle, StyleTable, StyleThead, StyleTbody, StyleTd, StyleTh, StyleTr } from "./style";
+import Button from "components/augmint-ui/button";
+
+const PAGING = 10;
 
 export class CustomTable extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            initData: this.props.data,
+            dataToShow: this.props.data.slice(0, PAGING),
+            buttonVisible: true
+        };
+        this.addData = this.addData.bind(this);
```

Comment:
let's not show the btn if ther's no transaction to show:
![image](https://user-images.githubusercontent.com/32713470/57310725-9078ba00-70ea-11e9-8037-93a1f8c0734b.png)